crates/evm/core/src/backend/fuzz.rs

```diff
@@ -168,7 +168,7 @@ impl<'a> DatabaseExt for FuzzBackendWrapper<'a> {
         transaction: B256,
         env: &mut Env,
         journaled_state: &mut JournaledState,
-        inspector: I,
+        inspector: &mut I,
```

Comment:
hmm, I don't think this is required because Inspector should be auto_impl for &mut Inspector

Reply:
used with `cx.state` (`state` is `&mut Cheatcodes`) throws "cannot move out of mutable ref..."
And I prefer this to using with double ref `&mut cx.state`

Reply:
gotcha